### DISCUSSION TITLE
Adding new posts in AstroPaper theme | Neko0xff tech blog

### DISCUSSION THREAD

**Discussion by giscus** [neko0xff/blog_astroV2#5] [Announcements]
# Adding new posts in AstroPaper theme | Neko0xff tech blog

Some rules & recommendations for creating or adding new posts using AstroPaper theme.

https://neko-0xff-blog.deno.dev/posts/adding-new-posts-in-astropaper-theme/

**Comment by neko0xff** [comment]
測試
